dipy/segment/bundles.py

```diff
@@ -612,7 +612,8 @@ def _prune_what_not_in_model(self, model_centroids,
         pruned_indices = [rtransf_cluster_map[i].indices
                           for i in np.where(mins != np.inf)[0]]
         pruned_indices = list(chain(*pruned_indices))
-        pruned_streamlines = transf_streamlines[np.array(pruned_indices)]
```

Comment:
OK - more feedback. This definitely raises an error when the pruned_indices are empty, because in that case, the array does not have integer dtype, so you get "indexing array must be of integer dtype". Should we raise another error in this case, or just check and return an empty list in that case?

Reply:
This we fixed by returning an empty object and printing this message: "You have removed all streamlines"

https://github.com/dipy/dipy/blob/7eca4013bd3a407df2fac533ba6cec3b310eb90f/dipy/segment/bundles.py#L829